README.md

```diff
@@ -7,16 +7,19 @@
     [![codecov](https://codecov.io/gh/ballerina-platform/ballerina-lang/branch/master/graph/badge.svg)](https://codecov.io/gh/ballerina-platform/ballerina-lang)
   [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
 
-Ballerina is a compiled, type safe, concurrent programming language designed to make it simple to write microservices that integrate APIs.
+In the cloud, applications interact with their environment over the network. Ballerina is a general-purpose programming language with features optimized for these network distributed applications. It is easy to write and modify and is suitable for application programmers.
```

Comment:
```suggestion
In the cloud, applications interact with their environment over the network. Ballerina is a general-purpose programming language with features optimized for these network-distributed applications. It is easy to write and modify and is suitable for application programmers.
```

Reply:
I consulted Rebecca and she said it can be written both ways. Since our tagline and most materials say network distributed (without the hyphen), shall we keep as is?

Reply:
You can write it in both ways when the term "network-distributed" is used on its own. Here, since it is used as an adjective to describe "applications", we need to say "network-distributed applications."

Anyway, I am -1 to use it as our tagline as it linguistically doesn't sound right to say "network distributed". We do get such a word even when we do a Google search. The correct term is "distributed networks".  I doubt whether it should be "distributed-network applications". To be on the safe side, we can say "applications distributed over a network" to avoid distorting the actual meaning.